apps/main/src/loan/entities/appstats-daily-volume.ts

```diff
@@ -0,0 +1,15 @@
+import { queryFactory } from '@ui-kit/lib/model/query'
+import { EmptyValidationSuite } from '@ui-kit/lib'
+import { httpFetcher } from '@/loan/utils/helpers'
```

Comment:
`export const httpFetcher = (uri: string) => fetch(uri).then((res) => res.json())`

Not sure why this helper even exists tbh 😅